tensorflow/core/debug/debug_grpc_io_utils_test.cc

```diff
@@ -25,6 +25,9 @@ limitations under the License.
 #include "tensorflow/core/platform/env.h"
 #include "tensorflow/core/util/event.pb.h"
 
+// TODO(cais): Support grpc:// debug URLs in open source once Python grpc
+//   genrule becomes available. See b/23796275.
+#if defined(PLATFORM_GOOGLE)
```

Comment:
Yep, that sounds like the best plan! (I don't know if we have a policy on `#if PLATFORM_GOOGLE` code in OSS, but it seems like it'd be best to minimize it.)